src/histolab/tile.py

```diff
@@ -0,0 +1,162 @@
+import os
+from pathlib import Path
+
+import numpy as np
+from PIL import Image
+
+from .filters import image_filters as imf
+from .filters import morphological_filters as mof
+from .filters.util import mask_percent
+from .types import CoordinatePair
+from .util import lazyproperty
+
+
+class Tile:
+    def __init__(self, image: Image.Image, coords: CoordinatePair, level: int = 0):
+        self._image = image
+        self._level = level
+        self._coords = coords
+
+    @lazyproperty
```

Comment:
why this property to return an attribute of the class? We do already have self._image without this prop

Reply:
Yes but is a way to expose `image` as a read-only attribute, right?

Reply:
oh ok i didn't realize we wanna have that as a read-only attr

Reply:
yeah, Tile is a read-only container to me